tcms/testcases/templates/testcases/get.html

```diff
@@ -83,6 +83,21 @@ <h2 class="card-pf-title" style="text-align: left">
                 <span class="fa fa-calendar"></span>{{ object.create_date }}
             </h2>
 
+            <h2 class="card-pf-title" style="text-align: left">
+                <span class="fa fa-clock-o"></span>{% trans 'Setup duration' %}:
+                {{ object.setup_duration }}
+            </h2>
+
+            <h2 class="card-pf-title" style="text-align: left">
+                <span class="fa fa-clock-o"></span>{% trans 'Testing duration' %}:
+                {{ object.testing_duration }}
+            </h2>
+
+            <h2 class="card-pf-title" style="text-align: left">
+                <span class="fa fa-clock-o"></span>{% trans 'Expected duration' %}:
+                {{ object.expected_duration }}
```

Comment:
Opening the page without assigning any values to these new fields leads to:

`TypeError: unsupported operand type(s) for +: 'NoneType' and 'NoneType'`